docs/se/tracing.adoc

```diff
@@ -52,7 +52,51 @@ include::{rootdir}/includes/dependencies.adoc[]
 ----
 <dependency>
     <groupId>io.helidon.tracing</groupId>
-    <artifactId>helidon-tracing</artifactId>
+    <artifactId>helidon-tracing</artifactId>    <1>
+</dependency>
+    <dependency>
+        <groupId>io.helidon.webserver.observe</groupId>
+        <artifactId>helidon-webserver-observe-tracing</artifactId> <2>
+    </dependency>
+----
+<1> Helidon tracing dependency.
+<2> Observability dependencies for tracing.
+
+For further processing of the tracing data, different providers are used.
+
+For Jaeger:
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.tracing.providers</groupId>
+    <artifactId>helidon-tracing-providers-jaeger</artifactId>
+</dependency>
+----
+
+For Zipkin:
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.tracing.providers</groupId>
+    <artifactId>helidon-tracing-providers-zipkin</artifactId>
+</dependency>
+----
+
+For OpenTelemetry:
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.tracing.providers</groupId>
+    <artifactId>helidon-tracing-providers-jaeger</artifactId>
```

Comment:
The OTel case should not refer to the Jaeger provider, should it?

Reply:
Fixed :)